utils/eventtracking.js

```diff
@@ -1,13 +1,15 @@
-const MIXPANEL_TOKEN = 'e98aa9d6d259d9d78f20cb05cb54f5cb';
+const SEGMENT_WRITE_KEY = 'vErt52x0024wPxUfLDZIKcYRPQgh6NyT';
```

Comment:
Using my personal key for testing, but I need an official NEAR key before this is merged. Not sure who owns the Segment account.

CC @AustinBaggio @TiffanyGYJ

Reply:
```suggestion
const SEGMENT_WRITE_KEY = 'ePJM5UVxZGUxLP9dH2wGDvUE7hC2MbTX';
```